docs/guide/technology-choices/load-balancing-overview-content.md

```diff
@@ -25,7 +25,7 @@ The following table summarizes the Azure load-balancing services.
 | Azure Front Door          | Global          | HTTP(S)             |
 | Azure Traffic Manager     | Global          | Non-HTTP(S)         |
 | Azure Application Gateway | Regional        | HTTP(S)             |
-| Azure Load Balancer       | Regional        | Non-HTTP(S)         |
+| Azure Load Balancer       | Regional or Global | Non-HTTP(S)         |
 
 ## Azure load-balancing services
```

Comment:
Please also update the text below that says "Load Balancer support for cross-region load balancing is in preview. For more information, see..." Review the "Load Balancer" bullet point and update it to reflect the new situation.  Thanks.

Reply:
Not quite what I had in mind, sorry for not being clearer.  I pushed an update.